styles/Home.module.css

```diff
@@ -0,0 +1,116 @@
+.container {
```

Comment:
Let's have a team discussion on how we want to use CSS. There are couple of options
- CSS Module (this pull request)
- Tailwind
- CSS in JS ( multiple options such as styled-components, emotion, styled-jsx etc)